### DISCUSSION TITLE
Run workflow on push (direct to branch or pull request) but avoid running when tags are created

### DISCUSSION THREAD

**Discussion by enen92** [community/community#25121] [Actions]
<p>I have two workflows:</p>
<p>a) Workflow 1 runs whenever a push is made to the repository either when someone pushes directly to the branch or when someone sends a pull request to the branch:</p>
<pre><code>name: Workflow1

on: [push, pull_request]

jobs:
  some checks:
      ....
</code></pre>
<p>b) Workflow 2 runs only when tags are pushed (and does a deploy)</p>
<pre><code>name: Workflow 2

on:
  create:
    tags:
      - v*

jobs:
  deploy job:
    ...
</code></pre>
<p>Due to the way the workflows are written, workflow 1 fails to run when tagging. Is there any way to modify the workflow1 so that it doesn’t run when tags are created but is still ran when pushes or PRs are made?</p>
<p>Thanks in advance</p>

**Comment by enen92** [comment, marked as answer]
<p>ok solved this by using </p>
<pre><code>on:
    push:
      branches:
        - '*'
      tags-ignore:
        - v*
    pull_request:
      branches:
        - '*'
</code></pre>

**Comment by nxtshield** [comment]


**Reply by nxtshield** [reply]
Respone #2
